### PR TITLE
Blake Nedved - Add support for Google Analytics

### PR DESCRIPTION
**Pull request by nedvedba**
# Description

Adds the ability to add google analytics metrics tracking to the web UI.

* [x] Changelog comment added
* [x] Feature description provided
* [x] Labels added

### CODE REVIEW

File: scripts/generate_datafed.sh [hunk 1]
```diff
@@ -206,6 +213,8 @@ export DATAFED_WEB_CERT_PATH="$local_DATAFED_WEB_CERT_PATH"
 export DATAFED_WEB_USER=""
 # How the web server communicates with the core server, assumes an internal network
 export DATAFED_CORE_ADDRESS_PORT_INTERNAL="$local_DATAFED_CORE_ADDRESS_PORT_INTERNAL"
+# The id for the associated Google Analytics tag, if left empty, Google Analytics will be disabled
+export DATAFED_GOOGLE_ANALYTICS_TAG="$local_DATAFED_GOOGLE_ANALYTICS_TAG"
```
Comment (JoshuaSBrown) [inline scripts/generate_datafed.sh:217]: We probably need a better description of where this comes from. Where in google analytics do you get it from? Maybe an example of what one might look like.

File: web/datafed-ws.js [hunk 2]
```diff
@@ -66,7 +66,8 @@ var g_host,
     g_ver_api_major,
     g_ver_api_minor,
     g_ver_api_patch,
-    g_tls;
+    g_tls,
+    g_ga;
```
Comment (JoshuaSBrown) [inline web/datafed-ws.js:70]: I prefer more descriptive names then ga, maybe goog_analy, or google_analytics. My approach is if I can open a file and know what something is it is a good name. ga is a bit ambiguous.

File: scripts/generate_ws_config.sh [hunk 3]
```diff
@@ -227,6 +240,10 @@ client_secret=${local_DATAFED_GLOBUS_APP_SECRET}
 # This is the address to talk with the core server which is listening on 
 # port 7513, assuming internal network.
 server_address=tcp://${local_DATAFED_CORE_ADDRESS_PORT_INTERNAL}
+
+[operations]
+# This is the tag associated with a Google Analytics installation that metrics will be sent to.
+google_analytics_tag=${local_DATAFED_GOOGLE_ANALYTICS_TAG}
```
Comment (JoshuaSBrown) [inline scripts/generate_ws_config.sh:246]: I like that this is in an operations section, why did you pick operations instead web?

**Review by JoshuaSBrown — CHANGES_REQUESTED** [review]
Just make the g_ga variable more descriptive and then this will be good to go!

File: scripts/generate_ws_config.sh (hunk 3 above)
Comment (nedvedba) [reply scripts/generate_ws_config.sh:246]: Well that entire file is configuration for the web server so I thought a new operations category was more descriptive to what this configuration option was for.

**Comment by JoshuaSBrown** [comment]
@nedvedba one final comment please add an item to the changelog.

**Comment by JoshuaSBrown** [comment]
In the future, make sure you add corresponding labels. Other than that, this is great.

**Comment by JoshuaSBrown** [comment]
@par-hermes format

**Review by JoshuaSBrown — APPROVED** [review]

**Pull request merged**